lib/include/dlisio/stream.hpp

```diff
@@ -17,8 +17,39 @@ class stream {
     explicit stream( lfp_protocol* p ) noexcept (true) : f(p) {};
 
     void seek( std::int64_t offset ) noexcept (false);
-    std::int64_t tell() const noexcept(true);
-    std::int64_t absolute_tell() const noexcept(false);
+
+    /** Physical tell
+     *
+     * The tell reported by the inner-most lfp_protocol. I.e. the actual number
+     * of bytes into the file as the file is seen on disk */
+    std::int64_t ptell() const noexcept(false);
+
+    /** Logical tell
+     *
+     * logical tell is the tell reported by current (outer-most) lfp_protocol.
+     *
+     * Example
+     * -------
+     *
+     * A file as seen on disk
+     *
+     *                                           (EOF)
+     * byte offset      0    100      300   400   500
+     *                   --------------------------
+     *                  | ... | header | ... | ... |
+     *                   --------------------------
+     *                        ^                 ^
+     *                        |                 |
+     *                  lfp_protocol         current
+     *                     opened            position
+     *                      here             in file
+     *
+     * In the above example where the current protocol was open at ptell=100
+     * and that the 200 bytes corresponding to some header are abstracted away
+     * by the lfp_protocol, and the current position in the file is as marked.
+     * Then ltell == 150 and ptell = The ltell
```

Comment:
1. ptell = ??
2. Actually the paragraph structure is weird:
    In the above example where X and that Y, and Z. Then RESULT.
  More sense would be:
   In the above example X, Y, and Z. This makes RESULT.